suite-native/device-manager/src/components/DeviceItemContent.tsx

```diff
@@ -1,82 +1,201 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { useSelector } from 'react-redux';
+import { View } from 'react-native';
 
-import { Icon, IconName } from '@suite-common/icons';
+import { Icon, IconSize } from '@suite-common/icons';
 import { HStack, Box, Text } from '@suite-native/atoms';
-import { Translation } from '@suite-native/intl';
+import { Translation, useTranslate } from '@suite-native/intl';
 import {
     selectDeviceNameById,
     DeviceRootState,
     PORTFOLIO_TRACKER_DEVICE_ID,
     selectDeviceLabelById,
 } from '@suite-common/wallet-core';
-import { TrezorDevice } from '@suite-common/suite-types';
+import { AcquiredDevice } from '@suite-common/suite-types';
 import { TypographyStyle } from '@trezor/theme';
-import { useActiveColorScheme } from '@suite-native/theme';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { DeviceModelIcon } from './DeviceModelIcon';
+
+type DeviceItemContentVariant = 'simple' | 'walletDetail';
 
 type DeviceItemContentProps = {
-    deviceId?: TrezorDevice['id'];
+    device?: AcquiredDevice;
```

Comment:
Why did you change this? Prefer sending ID as a prop and use selector here to select the device

Reply:
Selecting by id can return different device than expected. This is because all passphrased devices have the same device id as the non passphrased device for which you opened the passphrase.

Reply:
Oh, we should not selecting by device.id in this case, we should use device.state which is unique per wallet

Reply:
b11b9d623c